OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/ApplicationControllerTests.cs

```diff
@@ -82,7 +82,7 @@ public void Setup()
         applications = ApplicationDTOsGenerator.Generate(2).WithWorkshopCard(workshops.First()).WithParent(parent);
     }
 
-    
+
```

Comment:
(Code style) Please avoid adding extra new lines. 1 empty line is sufficient spacer.

Reply:
My bad, fixed.